content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -78,55 +78,56 @@ The latency, throughput, and the proxies' CPU and memory consumption are measure
 ### Sidecar CPU and memory usage
 
 Since the sidecar proxy performs additional work on the data path, it consumes CPU
-and memory. In Istio 1.22, a sidecar proxy consumes about 0.5 vCPU per 1000
-requests per second.
+and memory. In Istio 1.24, with 1000 http requests per second containing 1 KB of payload each
+- a sidecar proxy with 4 worker threads consumes about 27% CPU and 61 MB of memory.
+- a waypoint proxy with 2 worker threads consumes about 41% CPU and 57 MB of memory
+- a ztunnel proxy consumes about 7% CPU and 12 MB of memory.
 
 The memory consumption of the proxy depends on the total configuration state the proxy holds.
 A large number of listeners, clusters, and routes can increase memory usage.
-In a large namespace with [namespace isolation](/docs/reference/config/networking/sidecar/) enabled, the proxy consumes approximately 50 MB of memory.
 
 Since the proxy normally doesn't buffer the data passing through,
 request rate doesn't affect the memory consumption.
 
 ### Latency
 
-Since Istio injects a sidecar proxy on the data path, latency is an important
+Since Istio adds a sidecar proxy or ztunnel proxy on the data path, latency is an important
 consideration.
 Every feature Istio adds also adds to the path length inside the proxy and potentially affects latency.
 
 The Envoy proxy collects raw telemetry data after a response is sent to the
-client. The time spent collecting raw telemetry for a request does not contribute
-to the total time taken to complete that request. However, since the worker
-is busy handling the request, the worker won't start handling the next request
-immediately. This process adds to the queue wait time of the next request and affects
-average and tail latencies. The actual tail latency depends on the traffic pattern.
+client.
+The time spent collecting raw telemetry for a request does not contribute to the total time taken to complete that request.
```

Comment:
Arguably this isn't true for ztunnel

Reply:
But that is pretty pedantic, I don't mind if it stays as-is

Reply:
I think this was a formatting change from my part, so I will keep as is.